packages/generator/src/utils/getNextPackageVersion.ts

```diff
@@ -1,14 +1,20 @@
-import fs from 'fs';
+import fsExtra from 'fs-extra';
 import path from 'path';
 import semver from 'semver';
 
 import { PackageVersion } from '../constants';
 
-export function getNextPackageVersion(packageDir: string, options?: { patch?: boolean }) {
-  if (!options || !options.patch || !fs.existsSync(packageDir)) {
+export async function getNextPackageVersion(packageDir: string, options?: { patch?: boolean }) {
```

Comment:
I was thinking that `regenerate-all` should always sync package versions across the board.

How about we instead enforce that whenever we `regenerate-all` we need to bump the minor version of the root `package.json`. Then it's pretty easy to keep track as we'll know patches are individual releases and minors are across the board ones.

Reply:
Good idea, we should start doing that!